CONTRIBUTING.md

```diff
@@ -35,6 +39,7 @@ we have pre-commit hooks for Black, Flake8, and Isort configured in the versions
 pip install pre-commit
 pre-commit install
 ```
+in you local project root folder.
```

Comment:
```suggestion
in you local project root folder, if you want to use `pre-commit` else you can just run `pytest tests`.
```

Reply:
I agree that we should emphasize that the hooks are not mandatory.